scripts/rebase-ledger/index.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
This is a really minor nit on conventions. Does the concept of `index.*` exist outside of nodeJS/web development? It feels like a mixing of paradigms to name a shell file `index.sh` but at the same time I think the intention is well-communicated through the choice of name so I'm inclined to let this slide.

Reply:
🏴